app/Http/Controllers/ApplicationController.php

```diff
@@ -41,12 +42,37 @@ public function index()
     public function showWithJob(JobPoster $jobPoster, JobApplication $application)
     {
         if ($jobPoster->job_applications->contains($application)) {
+            // If the application is version two then show the version two application review page.
+            if ($application->version_id !== null && $application->version_id == 2) {
+                return $this->showVersionTwo($application);
+            }
             return $this->show($application);
         } else {
             return abort(404);
         }
     }
 
+    /**
+     * Display version two application.
```

Comment:
Since this method always returns the manager view, with `show_review = true`, maybe reference that in the method name or in the description. Something as small as "Display version two of application, from Manager perspective, with review toolbar included."